frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -199,8 +199,14 @@ export const ExpandablePolicyCard = ({
   /**
    * Displays the actions
    */
-  const displayActions = policyRule.actions.map((a, i) => {
-    return <ActionAndSubjectListItem key={i} title={a} onRemove={() => handleRemoveAction(i, a)} />;
+  const displayActions = policyRule.actions.map((actionId, i) => {
+    return (
+      <ActionAndSubjectListItem
+        key={i}
```

Comment:
We should avoid using index as a key in lists. You can read more about it [here](https://react.dev/learn/rendering-lists). React actually uses index as the default mechanism under the hood if no specific keys are provided.
```suggestion
  const displayActions = policyRule.actions.map((actionId) => {
    return (
      <ActionAndSubjectListItem
        key={actionId}
```

Reply:
I agree, that was from the old code, and I did not spot it. Fixed now 😄